.travis.yml

```diff
@@ -20,7 +20,8 @@ script:
   - |
       echo "Generating service crates..." &&
       ( cd service_crategen && \
-        cargo run -- generate -c ./services.json -o ../rusoto/services && \
+        rustup install nightly && \
```

Comment:
I'm not a travis expert, and I don't think it makes that much of a difference, but wouldn't this be better as an `install` task?

Reply:
Changed it to be a `before_install` task!